libnd4j/tests_cpu/run_tests.sh

```diff
@@ -56,9 +64,134 @@ if [ -n "$BUILD_PATH" ]; then
     export PATH="$PATH:$BUILD_PATH"
 fi
 
+# DeclarableOpsTests14.Reshape20
+# DeclarableOpsTests14.Reshape17
+# DeclarableOpsTests14.Reshape9
 unameOut="$(uname)"
 echo "$OSTYPE"
+if [[ "$TEST_FILTER" != "none" ]]; then
+   export BLOCK_SIZE_SCALAR_SCAN=1
+   export GRID_SIZE_SCALAR_SCAN=1
+   export GRID_SIZE_TRANSFORM_SCAN=1
+   export BLOCK_SIZE_TRANSFORM_SCAN=1
+   export SHARED_MEM_SIZE_TRANSFORM_SCAN=256
+   export GRID_SIZE_COL2IM=256
+   export BLOCK_SIZE_COL2IM=256
+   export SHARED_MEM_SIZE_COL2IM=16000
+    export GRID_SIZE_IM2COL=256
+    export BLOCK_SIZE_IM2COL=256
+    export SHARED_MEM_SIZE_IM2COL=16000
+       export BLOCK_SIZE_RANDOM=128
+        export GRID_SIZE_RANDOM=128
+  export GRID_SIZE_POOLING=256
+  export BLOCK_SIZE_POOLING=256
+    export GRID_SIZE_MERGE=256
+      export   BLOCK_SIZE_MERGE=256
+       export  SHARED_MEM_SIZE_MERGE=256
+       export GRID_SIZE_DIAG_PART=128
+       export BLOCK_SIZE_DIAG_PART=128
+       export GRID_SIZE_SEGMENT_MEAN=128
+       export BLOCK_SIZE_SEGMENT_MEAN=128
+     export GRID_SIZE_CLIP=128
+             export BLOCK_SIZE_CLIP=128
+            export GRID_SIZE_SWAP_UNSAFE=128
+            export  BLOCK_SIZE_SWAP_UNSAFE=256
+            export GRID_SIZE_SEGMENT=128
+            export BLOCK_SIZE_SEGMENT=128
```

Comment:
Is there a reason why this is so randomly indented?

Reply:
Nah just no IDE auto formatting for bash and wanted to wait till I was done to fix. Done.